packages/styling/src/interfaces/ISemanticColors.ts

```diff
@@ -52,6 +52,11 @@ import { ISemanticTextColors } from './ISemanticTextColors';
  * Lists differ from menus in that they are designed to show infinite amounts of items, often scroll,
  * and have a large and complex interaction surface.
  * This category covers all kinds of lists, whether they're typical one-item-per-row lists (like DetailsList) or ones with a tiled layout.
+ *
+ * Note: We are aware that the usage of semantic slots vs. palette slots is not consistent across the Fabric controls code. Both
```

Comment:
we have `Note:` at the very top as well on Line 8. i think we should move this up

Reply:
moved to top now!